app/src/main/java/com/kks/nimblesurveyjetpackcompose/model/response/SurveyAttributesResponse.kt

```diff
@@ -0,0 +1,26 @@
+package com.kks.nimblesurveyjetpackcompose.model.response
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+data class SurveyAttributesResponse(
+    @Json(name = "title")
+    val title: String?,
+    @Json(name = "description")
+    val description: String?,
+    @Json(name = "thank_email_above_threshold")
+    val thankEmailAboveThreshold: String?,
+    @Json(name = "thank_email_below_threshold")
+    val thankEmailBelowThreshold: String?,
+    @Json(name = "is_active")
+    val isActive: Boolean?,
+    @Json(name = "cover_image_url")
+    val coverImageUrl: String?,
+    @Json(name = "created_at")
+    val createdAt: String?,
+    @Json(name = "active_at")
+    val activeAt: String?,
```

Comment:
Can they be converted to `Date`? Moshi should have support for the `Date` conversion from strings.

Reply:
Will be converted in [Home Screen PR](https://github.com/kaungkhantsoe/NimbleSurveyJetpackCompose/pull/74)